src/main/java/uk/gov/hmcts/reform/wataskmanagementapi/services/CamundaService.java

```diff
@@ -239,20 +239,30 @@ public void completeTask(String taskId,
             eventVariable.put("eventId", new CamundaVariable(searchEventAndCase.getEventId(), "String"));
             Map<String, Map<String, CamundaVariable>> dmnRequest = new HashMap<>();
             dmnRequest.put("variables", eventVariable);
-            // A List (Array) with a map (One object) with objects inside the object (String and CamundaVariable).
-            List<Map<String, CamundaVariable>> evaluateDmnResult =
-                camundaServiceApi.evaluateDMN(
-                    authTokenGenerator.generate(),
-                    getTableKey(searchEventAndCase.getCaseJurisdiction(), searchEventAndCase.getCaseType()),
-                    dmnRequest
-                );
+            List<Map<String, CamundaVariable>> evaluateDmnResult;
+            if(searchEventAndCase.getCaseJurisdiction().equals("ia") &&
+               searchEventAndCase.getCaseType().equals("asylum")) {
+                // A List (Array) with a map (One object) with objects inside the object (String and CamundaVariable).
+               evaluateDmnResult =
+                    camundaServiceApi.evaluateDMN(
+                        authTokenGenerator.generate(),
+                        getTableKey(searchEventAndCase.getCaseJurisdiction(), searchEventAndCase.getCaseType()),
+                        dmnRequest
+                    );
+            } else {
+                return Collections.emptyList();
+            }
+
+            if (evaluateDmnResult.isEmpty()) {
+                return Collections.emptyList();
+            }
 
             List<String> taskTypes = evaluateDmnResult.stream()
                 .map(result -> getVariableValue(result.get("task_type"), String.class))
                 .collect(Collectors.toList());
 
 
-            if (taskTypes.isEmpty()) {
+            if (taskTypes.isEmpty() || evaluateDmnResult == null) {
```

Comment:
Not sure if the `evaluateDmnResult` should be null here?  You are already calling `evaluateDmnResult.isEmpty()` in line 256 which would be an NPE if it's null?

Ideally we don't want any nulls in the code.  This is a List so if there are no results it should an empty list.